packages/contracts/docs/osx/01-how-it-works/01-core/02-permissions/index.md

```diff
@@ -7,29 +7,29 @@ title: Permissions
 At Aragon, we believe that **DAOs are permission management systems**.
 Permissions between contracts and wallets allow a DAO to manage and govern its actions.
 
-Here, you will learn how the permissions in aragonOSx work, how they can be granted and revoked from wallets and contracts, and how they are managed through the DAO.
+Here, you will learn how the permissions in Aragon OSx work, how they can be granted and revoked from wallets and contracts, and how they are managed through the DAO.
 
 As we mentioned earlier, it is essential that only the right person or contract can execute a certain action. As a developer, you might have seen or used [modifiers such as `onlyOwner`](https://docs.openzeppelin.com/contracts/2.x/api/ownership#Ownable) in contracts. This `onlyOwner` modifier provides basic access control to your DAO: only the `owner` address is permitted to execute the function to which the modifier is attached.
 
 In aragonOSx, we follow the same approach but provide more advanced functionality:
```

Comment:
Aragon OSx

Reply:
Fixed in 66151f518d01899579276fd6df43401e0a0e1d79